src/master/master.js

```diff
@@ -301,7 +302,7 @@ export class Master {
     this.transportAPI.send({
       playerID,
       type: 'sync',
-      args: [gameID, filteredState, log],
+      args: [gameID, filteredState, log, gameMetadata],
```

Comment:
We should scrub the credentials from `gameMetadata` before it is sent over.

Reply:
I overlooked this, you are right.